app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="crayons-card p-5">
+  <% if page.image %>
+    <div class="s:max-w-50 mb-4">
+      <img src="<%= page.image %>" height="auto" alt="Avatar">
+    </div>
+  <% end %>
+  <div class="color-primary">
+    <p class="fs-xl fw-bold lh-tight mb-2">
```

Comment:
Probably more semantically correct would be `<h2>` or `<h3>` instead of `<p>` for title.